src/pages/RecommendedProduct.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+// import { FaHeart, FaStar } from 'react-icons/fa';
+import { useAppDispatch, useAppSelector } from '../redux/hooks/hooks';
+import { getRecommendedProducts } from '../redux/features/recommendedProductSlice';
+// import ProductPageAddToCart from '../components/carts/ProductPageAddToCart';
+// import Button from '../components/buttons/Button';
+import { DynamicData } from '../@types/DynamicData';
+import RecommendedDesign from '../components/Recommend';
+
+interface RecommendedProps {
+	prodId: string;
+}
+
+const RecommendedProduct = ({ prodId }: RecommendedProps) => {
+	const dispatch = useAppDispatch();
+	const { products, isLoading, error } = useAppSelector(
+		(state) => state.recommendedProducts,
+	);
+
+	useEffect(() => {
+		if (prodId) {
+			dispatch(getRecommendedProducts(prodId)).unwrap();
+		}
+	}, [dispatch, prodId]);
+
+	return (
+		<>
+			<div className="text-2xl pb-2 mt-5">
+				<h2>You may also like:</h2>
```

Comment:
This could be visible only when there is a recommended products

Reply:
done